doc/example/model_selection.ipynb

```diff
@@ -331,18 +311,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "initial_model_label = {initial_model.get_hash(): initial_model.model_id}\n",
-    "\n",
-    "pvs.plot_calibrated_models_digraph(\n",
-    "    problem=pypesto_select_problem_2,\n",
-    "    labels={\n",
-    "        **get_digraph_labels(\n",
-    "            pypesto_select_problem_2.calibrated_models.values(),\n",
-    "            criterion=Criterion.AIC,\n",
-    "        ),\n",
-    "        **initial_model_label,\n",
-    "    },\n",
-    ");"
+    "plot_data = plot.PlotData(\n",
+    "    models=petab_select_problem.state.models,\n",
+    "    criterion=Criterion.AIC,\n",
+    ")\n",
+    "plot_data.augment_labels(criterion=True)\n",
+    "plot.graph_history(plot_data=plot_data);"
```

Comment:
I like this a lot, feels easier to pass arguments to plotting

Reply:
I'm glad, another benefit is that it simplified the plotting code in the library quite a bit too :grinning: